cfme/tests/intelligence/reports/test_reports_manual.py

```diff
@@ -525,3 +525,51 @@ def test_vm_volume_free_space_less_than_20_percent():
     Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1686281
     """
     pass
+
+
+@pytest.mark.manual
+@test_requirements.report
+@pytest.mark.tier(0)
+def test_reports_filter_content():
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        initialEstimate: 1/3h
+        setup:
+            1. Go to Cloud Intel -> Reports -> All Reports
+            2. Select a report and queue it, make sure it's not empty.
+        testSteps:
+            1. Add a filter.
+            2. Traverse through all the rows and check if the content is filtered.
+        expectedResults:
+            1. Content must be filtered.
```

Comment:
it should be 
```
1.
2. Content must be filtered.
```
that way the expected result corresponds to the second step in polarion